app/views/content/fund-your-training/_questions.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="category__questions__container">
+
+  <section class="container category__questions">
+
+    <div class="category__question">
+      <h2>Want your own personal adviser?</h2>
+
+      <p>
+        Teacher training advisers provide free, one-to-one support by phone, text or email.
+        They can support you to find funding for your training.
+      </p>
+
+      <a href="/teacher-training-advisers" class="button">Get an adviser</a>
+    </div>
+
+    <div class="category__question">
+      <h2>Do you have a question about funding?</h2>
+
+      <p>
+        You can join one of our online question and answer sessions and ask an expert about what you could be eligible for.
+      </p>
+
+      <a href="/event-categories/online-q-as" class="button">Find out about our online events</a>
```

Comment:
```suggestion
      <a href="/event-categories/online-q-as" class="button">Find online events</a>
```
Would suggest reducing the copy so the green buttons are of a more equal size

Reply:
Agree

This link will need updating as the new events design will be in by then

Reply:
Also - there may be a gap when there are no online Q&A events - do we really want to limit ourselves here?